md/import-export-an-organization.md

```diff
@@ -1,31 +1,34 @@
-# Import/export an organization
+# Install/export an organization
 
-Logged on with the Administrator profile, you have rights to import or export an organization.
+This page explains what a user with the _Administrator_ profile in Bonita Portal or in the Bonita Administrator Application can see and do about the Organization of Bonita users (an .xml file). 
 
-## How to import an organization
+Here is a view of this page:
+![Install export organization](images/UI2021.1/install-export.png)<!--{.img-responsive}-->
 
-**Caution:**
-This will import a file containing your whole organization data. This organization data will be merged with existing data.
-In case of conflict, the priority is given to the data in the imported file. Take care not to overwrite information that has been updated in Bonita Portal.
 
-**Note:**
-In 6.3 there was a change to the structure of the .xml file. This means that you cannot import into Bonita Portal 6.3.0 or later an organization .xml file that was created in 6.2.x or earlier.
-You will need to first import the organization file into Bonita Studio and re-export it, so it is compatible.
+## How to install an organization
 
-To import an organization in .xml format, that has already been exported from Bonita Studio, containing your whole organization data.
+::: Warning
+This will import a file containing your whole organization data. This organization data will be merged with existing data.  
+In case of conflict, the priority is given to the data in the imported file. Take care not to overwrite information that has been updated in Bonita Portal since the last installation of the organization.
```

Comment:
```suggestion
In case of conflict, the priority is given to the data in the imported file. You should make sure not to overwrite information that has been updated in Bonita Portal since the last installation of the organization.
```